src/EventSubscriber/Init.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\password_enhancements\EventSubscriber;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\Core\Session\SessionManagerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\Url;
+use Drupal\password_enhancements\Controller\UserLogoutController;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Subscribes to Drupal initialization event.
+ */
+class Init implements EventSubscriberInterface {
```

Comment:
It is a "KernelRequestSubscriber" , there is no (hook) init :)

Reply:
I know, I'm just using it as one :) that's why I've used the name `Init`, but I can rename if you want.

Reply:
I would rather prefer if the class name would contain `Subscriber` and it would be `final`. If you also keep `Init` then we both win :)

Reply:
Ok :)